synfig-studio/src/gui/trees/layergrouptreestore.cpp

```diff
@@ -120,6 +116,22 @@ LayerGroupTreeStore::create(etl::loose_handle<synfigapp::CanvasInterface> canvas
 	return Glib::RefPtr<LayerGroupTreeStore>(new LayerGroupTreeStore(canvas_interface_));
 }
 
+// TODO(ice0): duplicated code
+template<typename T>
+static void set_gvalue_tpl(Glib::ValueBase& value, const T &v, bool use_assign_operator)
+{
+	Glib::Value<T> x;
+	g_value_init(x.gobj(), x.value_type());
```

Comment:
as you used gtkmm in the line below (`x.set()`), what about use it here too (`x.init()` as in previous code)?

Reply:
@ice0 ping

Reply:
Done